access/jira/README.md

```diff
@@ -66,6 +66,16 @@ server & manage access-requests. Use `tctl auth sign --format=tls` to generate
 the required PEM files, and make sure that the Auth Server's GRPC API is
 accessible at the address indicated by `auth_server`.
 
+`auth_server` can also be a proxy (e.g. when using a plugin with Teleport Cloud),
```

Comment:
You could avoid adding this explanation (here and in #220) by making `file` format and `IdentityCredentials` the default/recommended credential method. You could leave the `tls` format and `KeyPairCredentials` just for backwards compatiblity.